content/en/tracing/trace_collection/tracing_naming_convention/_index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Span Tags Semantics
+title: Span Tag Semantics
 kind: documentation
 further_reading:
     - link: 'logs/log_configuration/attributes_naming_convention'
```

Comment:
Probably worth adding a link to the naming convention in the further reading links